l1-contracts/contracts/common/interfaces/IBatchAggregator.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.24;
+
+interface IBatchAggregator {
+    function commitBatch(
+        bytes calldata _totalL2ToL1PubdataAndStateDiffs,
```

Comment:
why do we need to send both compressed diffs and their uncompressed version? I think we should decide on what we want. I would rather send only uncompressed ones